src/plugins/intel_gpu/src/graph/impls/onednn/fully_connected_onednn.cpp

```diff
@@ -130,25 +130,79 @@ struct fully_connected_onednn : typed_primitive_onednn_impl<fully_connected> {
         get_matmul_primitive_descriptor(const kernel_impl_params& impl_params,
                                         cldnn::engine& engine,
                                         size_t prim_input_size,
+                                        size_t prim_weights_rank,
                                         bool has_bias,
                                         const dnnl::primitive_attr& attr = dnnl::primitive_attr()) {
         auto input_layout = impl_params.get_input_layout(0);
         auto weights_layout = impl_params.get_input_layout(1);
         auto output_layout = impl_params.get_output_layout();
 
-        transform_layouts(input_layout, weights_layout, output_layout, prim_input_size);
+        dnnl::memory::format_tag target_fmt;
+        dnnl::memory::format_tag weights_fmt;
+
+        if (prim_input_size == 3) {
+            target_fmt = dnnl::memory::format_tag::abc;
+            weights_fmt = dnnl::memory::format_tag::acb;
+        } else if (prim_input_size == 4) {
+            target_fmt = dnnl::memory::format_tag::abcd;
+            weights_fmt = dnnl::memory::format_tag::abdc;
```

Comment:
`dnnl::memory::format_tag::abdc, dnnl::memory::format_tag::abced, dnnl::memory::format_tag::abcdfe`  are used for weight_fmt but I can't find these format in layout_to_memory_desc. Shouldn't these also be added to the layout_to_memory_desc?